index.bs

```diff
@@ -1034,8 +1046,8 @@ associated.
 </xmp>
 <div dfn-type="dict-member" dfn-for="PublicKeyCredentialEntity">
     :   <dfn>id</dfn>
-    ::  A unique identifier for the entity. This will be the [=ASCII serialization of an origin=] for a [=[RP]=],
-        and an arbitrary string specified by the [=[RP]=] for user accounts.
+    ::  A unique identifier for the entity. For a [=relying party=] entity, this will be the [=RP ID=]. For a user account
+        entity, this will be an arbitrary string specified by the [=relying party=].
```

Comment:
Try to say how this string is used, although probably in a different PR since it's not related to cleaning up RP IDs.